// Proto file representing references to EDB Rlc classes

syntax = "proto3";

package ansys.api.edb.v1;

import "edb_messages.proto";

message RlcMessage {
  ValueMessage r = 1;
  ValueMessage l = 2;
  ValueMessage c = 3;
  google.protobuf.BoolValue r_enabled = 4;
  google.protobuf.BoolValue l_enabled = 5;
  google.protobuf.BoolValue c_enabled = 6;
  google.protobuf.BoolValue is_parallel = 7;
}
